### ISSUE TITLE
Exception in receiveRecover of the ProcessIndex actor

### ISSUE THREAD

**Issue by merlijn** [ing-bank/baker#133]
Exception in receiveRecover

Version 1.3.3
```
ERROR akka.event.slf4j.Slf4jLogger$$anonfun$receive$1$$anonfun$applyOrElse$1 Exception in receiveRecover when replaying event type [com.ing.baker.runtime.actor.process_index.ProcessIndex$ActorDeleted] with sequence number [510] for persistenceId [index-29].

java.util.NoSuchElementException: key not found: fa0dcd27-1718-4ad9-a8b0-b21f9adae28f
```

**Comment by merlijn** [comment]
A workaround was released in `1.3.4`. 

This occurs when an `ActorDeleted` message was persisted without an `ActorCreated` message.

The real cause needs to be investigated.

**Comment by merlijn** [comment]
The real cause appears to have been split brain. Not a bug in the code.

With the default akka recover strategy. Events from the 'old' journal writer are ignored making it possible to receive an `ActorDeleted` before an `ActorCreated` event.
